spec/controllers/submissions_controller_spec.rb

```diff
@@ -797,10 +797,13 @@
         end
 
         it 'should collect all groupings when override is true' do
+          enqueuing_user = User.find_by user_name: @instructor.user_name
```

Comment:
I don't think the changes in this file are necessary? These values should be optional and not affect the rest of the functionality.

Reply:
The collect submissions controller method currently passes the enqueuing user to the submissionsjob by default, causing the tests to expect that option in the job to be set. The changes are to account for that.

Reply:
I'm not sure what you mean by "causing the tests to expect that option in the job to be set". If you're getting an error when you don't include this line, you can show that.